### PR TITLE
Shoutouts Week 28

### PR DESCRIPTION
**Pull request by gr2m**
I think like I missed what else happened in the past 4 weeks that we should give special thanks to ... any thoughts?

Preview: https://github.com/hoodiehq/hood.ie/blob/shoutout-week-28/_posts/2016-07-13-shoutouts-week-28.md

### CODE REVIEW

**Comment by gr2m** [comment]
ping @hoodiehq/maintainers

**Comment by jennwrites** [comment]
LGTM! 

My only suggestion would be to include some sort of headline beyond "Shoutouts". For example, "Shoutouts Week 24: Welcoming new contributors and backers" or some sort of recap statement to draw people in. 

Thanks for doing this @gr2m ❤️

**Comment by gr2m** [comment]
renamed title to `Shoutouts Week 28 – Welcoming Backers & Sponsors`, cool?

**Comment by jennwrites** [comment]
Works for me!

File: _posts/2016-07-13-shoutouts-week-28.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+We do our best to follow a more regular schedule with our shout out blog posts, but we could use some help. :)
+We started [documenting the workflow to create a blog post like this](https://github.com/hoodiehq/editorial/wiki/How-to-create-a-shout-out-blog-post), it’s about two hours of work right now. If you know JavaScript / Node.js, you can also help us by improving our [github-shoutouts](https://github.com/finnp/github-shoutouts) CLI tool. Please [get in touch](http://hood.ie/contact/).
+
+If you are interested, [there are many ways you can contribute](http://hood.ie/contribute/). <3
+
+### Backers and Sponsors
+
+Starting this month, you can contribute to Hoodie by becoming a personal backer or official sponsor on [Hoodie’s OpenCollective Page](https://opencollective.com/hoodie).
+
+We’d like to thank very first backers:
```
Comment (varjmes) [inline _posts/2016-07-13-shoutouts-week-28.md]: Should be "We'd like to thank our very first backers:"

File: _posts/2016-07-13-shoutouts-week-28.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+We also welcome
+
+- [doplumi](https://github.com/doplumi) contributed to improvements to [Hoodie’s documentation](http://docs.hood.ie/): [Avoid backticks for single line code.](https://github.com/hoodiehq/documentation/pull/223) and [Fix npm install command](https://github.com/hoodiehq/documentation/pull/222)
+ worked on [hoodiehq/documentation](https://github.com/hoodiehq/documentation)
+- [sthodup1](https://github.com/sthodup1) contributed to [Hoodie’s Camp Tutorial](https://camp-tutorial.hood.ie/): [Update 2.html with typo fix](https://github.com/hoodiehq/hoodie-camp-tutorial/pull/14)
+
+### New Contributions to Hoodie
+
+- [Aaron](https://github.com/courajs) continued to work on the new Admin Dashboard. He landed [Allow showing the password being typed in the new user form](https://github.com/hoodiehq/hoodie-admin/pull/11) and [Fix a silly-looking layout of the new-user dialog](https://github.com/hoodiehq/hoodie-admin/pull/12).
+- [Charlotte](https://github.com/Charlotteis) landed [fix(slack): make the slack form accessible](https://github.com/hoodiehq/hood.ie/pull/188)
```
Comment (varjmes) [inline _posts/2016-07-13-shoutouts-week-28.md:46]: This one was on the shelf for a long time, so happy it was merged!!

**Comment by varjmes** [comment]
just one typo!

**Comment by gr2m** [comment]
fixed, thanks! I guess this is ready then? The shoutouts are a bit outdated now, but c’est la vie! ¯\_(ツ)_/¯

**Pull request merged**